.travis.yml

```diff
@@ -15,3 +15,6 @@ script:
   - wcf
   - wcf start --author test --copyrightOwner test --license MIT
   - wcf factory --humanName test --description test --name test --orgNpm @test --orgGit test --gitRepo git@github.com:test/test.git
+  - yes n | wcf element --customElementTemplate Slim --name slim-test --description test
```

Comment:
I'd prefer to use `--no-useSass --no-addProps` but inquirer doesn't recognize those values are set, and prompts anyway, causing CI to fail.
Which is confusing since the help text documents using `--no` options

```
OPTIONS
  -c, --customElementTemplate=customElementTemplate  Custom element template to use
  -d, --description=description                      Description of the element
  -f, --factory=factory                              Factory to use
  -h, --help                                         show CLI help
  -h, --[no-]useHAX                                  Use HAX
  -n, --name=name                                    Name of the element
  -p, --[no-]addProps                                Add properties
  -s, --[no-]useSass                                 Description of the element
```

Reply:
Found work around adding a type check